task/upload.go

```diff
@@ -164,27 +163,14 @@ func getFileUrlForUploadTask(os *api.ObjectStore, cfg ImportTaskConfig, params a
 		u.Path = path.Join(u.Path, params.UploadedObjectKey)
 		return u.String(), nil
 	}
-	return getFileUrl(cfg, params.URL)
+	return getFileUrl(params.URL)
```

Comment:
The caller here also has a `cfg` param which is now unused without this. Can you propagate the arg removal for cleanliness?